docs/foundations/oidc-cheet-sheet.mdx

```diff
@@ -0,0 +1,109 @@
+
+```mermaid
+sequenceDiagram
+    participant user as User
+    participant frontend as Your Application
+    participant bi as OIDC Identity Provider (IdP)
+    user ->> frontend: click login
+    frontend->>bi: send authorization request to /authorize endpoint
+    bi->>user: ask user for authorization
+    user->>bi: user confirms
+	bi->>frontend: receive authorization code
+    frontend->>bi: send authorization code to /token endpoint
+    bi->>frontend: receive ID Token + Access Token
+    frontend->>user: logged in
+```
+
+Here's a step-by-step breakdown of the flow:
+
+1. The user initiates the login process by clicking on the login button within your application.
+1. Your application sends an authorization request to the OIDC Identity Provider by making an HTTP request to the /authorize endpoint.
+1. The OIDC Identity Provider responds by asking the user for authorization. The user confirms their identity.
+1. Upon successful confirmation, the OIDC Identity Provider issues an authorization code to your application.
+1. Your application then exchanges this authorization code for both an ID Token and an Access Token by making another HTTP request to the /token endpoint of the identity provider.
+1. The OIDC Identity Provider responds by providing the ID Token and Access Token to your application.
+1. With these tokens, your application can now consider the user as logged in and provide access to the protected resources.
+
+## ID Tokens vs Access Tokens
+
+**Access Tokens** come from the [OAuth 2.0 framework](/docs/terms/oauth) and are used to access protected resources. [Access Tokens](/docs/terms/access-token.md) are used in API calls as [bearer tokens](/docs/terms/bearer-token.md). They do not guarantee that the user is currently authenticated.
+
+**ID Tokens** come from [OIDC](/docs/terms/oidc.md) and are used to prove that the user has been authenticated. [ID Tokens](/docs/terms/id-token.md) are always [JSON Web Tokens (JWT)](/docs/terms/jwt.md). When the payload of the JWT is inspected you will find claims, information about the user such as the user's name and email and the OIDC service. You can use this token to determine if the user is logged in.
+
+## What to do in your application
+
+The first step is to request authorization from the user. To do this you will need to create an authorization url for the user to click on. Your OIDC identity provider (IdP) should provide the authorization endpoint but you will need to add additional parameters to this URL:
+
+The authorization URL follows the format below:
+
+```http /authorize
+https://idp-auth-server/authorize
+?response_type=code
+&scope=openid
+&client_id=12345
+&redirect_uri=https%3A%2F%2Fyour-site.com
+```
+
+:::tip
+Other parameters such as **state**, **code_challenge_method** and **code_challenge** may be set for higher security, but the above are the minimum required.
```

Comment:
If you want to link to an example with these parameters, there is one here: /docs/create-api-token#authorization-code-with-pkce-confidential-client

Reply:
thanks!